docs/source/services/timelock_service/installation.rst

```diff
@@ -50,3 +70,39 @@ Timelock Server Installation
               "healthy": true
           }
       }
+
+Add client(s) to Timelock
+-------------------------
+
+8. Back up each client, following the steps declared in :ref:`backup-restore`.
+
+9. Configure each client to use Timelock.
+   Detailed documentation is :ref:`here <timelock-client-configuration>`.
+   You must remove any ``leader``, ``timestamp``, or ``lock`` blocks; the ``timelock`` block to add looks like this:
+
+.. code-block:: yaml
+
+   atlasdb:
+      timelock:
+        client: tom
+        serversList:
+          servers:
+            - palantir-1.com:8080
+            - palantir-2.com:8080
+            - palantir-3.com:8080
+          sslConfiguration:
+            trustStorePath: var/security/truststore.jks
+
+10. Configure Timelock to respond to your clients.
+    Add each client as a named entry in the ``clients`` block of your :ref:`timelock-server-configuration`.
+
+
+.. code-block:: yaml
+
+   clients:
+      - tom
+      - jerry
+
+11. Restart your Timelock cluster. Once again, a healthcheck (see step 7) may be run to confirm that Timelock restarted correctly.
+
+12. Restart each client.
```

Comment:
I'd like to include a section on migrations here (http://palantir.github.io/atlasdb/html/services/timelock_service/migration.html). If using Cassandra we're ok because the automatic migrations will do their work, but people using dbkvs will need to manually do the migration before restarting their clients.